contract/models/contract_modification.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ContractModification(models.Model):
+
+    _name = 'contract.modification'
+    _description = 'Contract Modification'
+    _order = 'date asc'
+
+    date = fields.Date(
+        required=True,
+        string='Date'
+    )
+    description = fields.Text(
```

Comment:
Doing this multi-line, will be ugly when rendering the mail template. Options:

1. Not being multi-line.
2. More elegant: render a table in the mail for displaying this.

Reply:
Option 2 it's a good idea.